roles/lib_openshift_api/library/oc_secrets.py

```diff
@@ -0,0 +1,360 @@
+#!/usr/bin/env python
+'''
+module for openshift cloud secrets
+'''
+#   Examples:
+#
+#  # to initiate and use /etc/origin/master/admin.kubeconfig file for auth
```

Comment:
Is this comment right? Or should it be to list all secrets for the default namespace?